tests/test_info_parser.py

```diff
@@ -36,6 +36,19 @@ def test_bad_package_parsing(self):
         with self.assertRaises(UnparsableNEVRAError):
             profile_parser._get_name_vra_from_string("this-will_not_parse")
 
+    def test_gpg_pubkey_parsing(self):
+        tests = tests = {
```

Comment:
looks like you have an extra `tests = ` here

Reply:
O_o  Good catch, thanks!